edx_proctoring/templates/timed_exam/ready_to_submit.html

```diff
@@ -2,24 +2,23 @@
 <div class="sequence timed-exam completed" data-exam-id="{{exam_id}}">
   <h3>
     {% blocktrans %}
-      Are you sure you want to end your timed exam?
+      Are you sure that you want to end your timed exam?
```

Comment:
This might make learners think they're canceling their exams. Can we change "end" to "complete" or "submit"?